sdk/eventhubs/azure_messaging_eventhubs_checkpointstore_blob/examples/checkpoint_store_basic.rs

```diff
@@ -4,6 +4,10 @@
 // cspell:ignore appender OTLP
 
 //! Basic example showing how to create and use a blob checkpoint store.
+//! This example demonstrates creating a `BlobCheckpointStore`, claiming ownership of a partition,
```

Comment:
Nit: this becomes one long sentence otherwise

```suggestion
//!
//! This example demonstrates creating a `BlobCheckpointStore`, claiming ownership of a partition,
```